ng generate universal (after adding pwa) update main.ts incorrectly causing syntax error

<!--
IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION YOUR ISSUE MIGHT BE CLOSED WITHOUT INVESTIGATING
-->
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Command (mark with an `x`)
```
- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [x] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Versions
<!--
Output from: `node --version`, `npm --version` and `ng --version`.
  Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra? High Sierra?)
-->
`node --version`
node v11.0.0

`npm --version`
npm 6.4.1

`ng --version`
Angular CLI: 7.2.1
Node: 11.0.0
OS: darwin x64
Angular: 7.2.0
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router
... service-worker

Package                           Version
@angular-devkit/architect         0.12.1
@angular-devkit/build-angular     0.12.1
@angular-devkit/build-optimizer   0.12.1
@angular-devkit/build-webpack     0.12.1
@angular-devkit/core              7.2.1
@angular-devkit/schematics        7.2.1
@angular/cli                      7.2.1
@angular/pwa                      0.12.1
@ngtools/webpack                  7.2.1
@schematics/angular               7.2.1
@schematics/update                0.12.1
rxjs                              6.3.3
typescript                        3.2.2
webpack                           4.23.1

`OS`
macOS High Sierra
### Repro steps
<!--
Simple steps to reproduce this bug.
Please include: commands run (incl args), packages added, related code changes.
A link to a sample repo would help too.
-->
1) check out https://github.com/Jun711/angular-app-shell/tree/master
Note that when u check out the above repo on master branch, it already has pwa added using `ng add @angular/pwa`. (the issue might occur only when ng generate universal is done after ng add @angular/pwa but I am not sure if that is the case)

2) on master branch, run `ng generate universal uni-app-shell --client-project angular-app-shell`
```
CREATE src/main.server.ts (220 bytes)
CREATE src/app/app.server.module.ts (318 bytes)
CREATE src/tsconfig.server.json (219 bytes)
UPDATE package.json (1461 bytes)
UPDATE angular.json (4794 bytes)
UPDATE src/main.ts (431 bytes)
UPDATE src/app/app.module.ts (649 bytes)
added 4 packages from 5 contributors in 21.384s
```

3) note the src/main.ts is updated with syntax error: 
The update was from
```
platformBrowserDynamic().bootstrapModule(AppModule)
  .catch(err => console.error(err));
```
to
```
document.addEventListener('DOMContentLoaded', () => {
  platformBrowserDynamic().bootstrapModule(AppModule)
  .catch(err => console.error(err));
);
```

It just needs an extra closing curly brace. 
```
document.addEventListener('DOMContentLoaded', () => {
  platformBrowserDynamic().bootstrapModule(AppModule)
  .catch(err => console.error(err));
});
```
I am not sure what source code to edit to fix that. If you could point me to it, I can do a pull request if necessary. 

### The log given by the failure when I ran `ng serve`
<!-- Normally this include a stack trace and some more information. -->

Date: 2019-01-09T20:25:21.997Z
Hash: 9e6345c3e1fcb145c2b1
Time: 2420ms
chunk {main} main.js, main.js.map (main) 1.86 kB [initial] [rendered]
chunk {polyfills} polyfills.js, polyfills.js.map (polyfills) 92.4 kB [initial] [rendered]
chunk {runtime} runtime.js, runtime.js.map (runtime) 6.08 kB [entry] [rendered]
chunk {styles} styles.js, styles.js.map (styles) 16.6 kB [initial] [rendered]
chunk {vendor} vendor.js, vendor.js.map (vendor) 322 kB [initial] [rendered]

ERROR in src/main.ts(14,1): error TS1128: Declaration or statement expected.

ℹ ｢wdm｣: Failed to compile.


### Desired functionality
<!--
What would like to see implemented?
What is the usecase?
-->
to have main.ts updated without any syntax error

### Mention any other details that might be useful
<!-- Please include a link to the repo if this is related to an OSS project. -->

--- a/packages/schematics/angular/universal/index.ts
+++ b/packages/schematics/angular/universal/index.ts
@@ -134,7 +134,7 @@ function wrapBootstrapCall(options: UniversalOptions): Rule {
     const triviaWidth = bootstrapCall.getLeadingTriviaWidth();
     const beforeText = `document.addEventListener('DOMContentLoaded', () => {\n`
       + ' '.repeat(triviaWidth > 2 ? triviaWidth + 1 : triviaWidth);
-    const afterText = `\n${triviaWidth > 2 ? ' '.repeat(triviaWidth - 1) : ''});`;
+    const afterText = `\n${triviaWidth > 2 ? ' '.repeat(triviaWidth - 1) : ''}});`;
 
     // in some cases we need to cater for a trailing semicolon such as;
     // bootstrap().catch(err => console.log(err));

--- a/packages/schematics/angular/universal/index_spec.ts
+++ b/packages/schematics/angular/universal/index_spec.ts
@@ -150,10 +150,11 @@ describe('Universal Schematic', () => {
     const tree = schematicRunner.runSchematic('universal', defaultOptions, appTree);
     const filePath = '/projects/bar/src/main.ts';
     const contents = tree.readContent(filePath);
-    expect(contents).toMatch(/document.addEventListener\('DOMContentLoaded', \(\) => {/);
+    expect(contents)
+      .toMatch(/document.addEventListener\('DOMContentLoaded', \(\) => {[\w\W]+;[\r\n]}\);/);
   });
 
-  it('should wrap the bootstrap decleration in a DOMContentLoaded event handler', () => {
+  it('should wrap the bootstrap declaration in a DOMContentLoaded event handler', () => {
     const filePath = '/projects/bar/src/main.ts';
     appTree.overwrite(
       filePath,

